etc/flameshot.profile

```diff
@@ -0,0 +1,38 @@
+# Firejail profile for flameshot
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/flameshot.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-interpreters.inc
+include /etc/firejail/disable-passwdmgr.inc
+include /etc/firejail/disable-programs.inc
+include /etc/firejail/disable-xdg.inc
+
+caps.drop all
+ipc-namespace
+netfilter
+no3d
+# nodbus
+nodvd
+nogroups
+nonewprivs
+noroot
+nosound
+notv
+novideo
+protocol unix,inet,inet6
```

Comment:
What happens with just `protocol unix`? This would disable internet access, is that something flameshot needs?

Alternatively, what if you removed the protocol line entirely and replaced `netfilter` with `net none`? Does it still work?

Reply:
Flameshot has the option to upload screenshots to https://imgur.com